bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -453,11 +453,15 @@ func (kfapp *coordinator) Delete(resources kftypes.ResourceEnum) error {
 
 	switch resources {
 	case kftypes.ALL:
-		if err := platform(); err != nil {
-			return err
+		if err := k8s(); err != nil {
+			return fmt.Errorf("error while deleting k8 resources, aborting deleting the platform. Error %v", err)
```

Comment:
could you use new `KfError` interface when adding new error message?

example:
https://github.com/kubeflow/kubeflow/blob/master/bootstrap/pkg/kfapp/gcp/gcp.go#L351